agnpy/compton/synchrotron_self_compton.py

```diff
@@ -33,6 +33,36 @@ def __init__(self, blob, ssa=False, integrator=np.trapz):
         self.ssa = ssa
         self.integrator = integrator
 
+    def electron_energy_loss_rate_thomson(self, gamma):
+        fixed = ((4 / 3) * sigma_T * c * self.blob.u_ph_synch).to("erg/s")
+        return fixed * gamma ** 2
+
+    def electron_energy_loss_rate(self, gamma):
+        epsilon0 = nu_to_epsilon_prime(nu_to_integrate, self.blob.z, self.blob.delta_D)
+        sed_flux_synch = Synchrotron(self.blob).evaluate_sed_flux(
+            epsilon0,
+            self.blob.z,
+            self.blob.d_L,
+            self.blob.delta_D,
+            self.blob.B,
+            self.blob.R_b,
+            self.blob.n_e,
+            *self.blob.n_e.parameters,
+            gamma=gamma
+        )
+
+        # differential energy density
+        u_epsilon0 = (3 * np.power(self.blob.d_L, 2) * sed_flux_synch) / (
+                c * np.power(self.blob.R_b, 2) * np.power(self.blob.delta_D, 4) * epsilon0
+        )
```

Comment:
I think you need an extra factor of 3/4 here, compere with this:
https://github.com/cosimoNigro/agnpy/blame/506c038fa751369e63e20a85a222558a8997b8d4/agnpy/compton/synchrotron_self_compton.py#L109-L111